packages/core/src/index.ts

```diff
@@ -4,6 +4,7 @@ export * as emitter from './emitter'
 export { AddEventListenerOptions, EventListenerOptions, EventTarget, Emitter } from './emitter'
 export * as observable from './observable'
 export { Observer, Subscription, Observable } from './observable'
+export * as interopObservable from './interop-observable'
```

Comment:
Why do you want to export this from the package root? This is an internal implementation detail

Reply:
Needed for `fp-ts` library. How we should handle it?

Reply:
Should import from `core/src/interop-observable`?